README.md

```diff
@@ -407,6 +416,10 @@ time                akey    bkey    ckey    priority rule      value
 
 ![opsgenie example](https://github.com/falcosecurity/falcosidekick/raw/master/imgs/opsgenie.png)
 
+### Discord
+
+![discord example](imgs/discord_example.png)
```

Comment:
```suggestion
![discord example](https://github.com/falcosecurity/falcosidekick/raw/master/imgs/discord_example.png)
```

This is mandatory to use the full URL for having images in dockerhub page

Reply:
Ah ok!